src/main/java/ch/njol/skript/events/EvtJump

```diff
@@ -0,0 +1,39 @@
+package ch.njol.skript.events;
```

Comment:
Why is this file not a `.java` ?
![image](https://user-images.githubusercontent.com/20037329/219000815-1ab734f8-52e2-4859-bc2c-89f21f2f665c.png)